bsp/esp32_s3_lcd_ev_board/idf_component.yml

```diff
@@ -1,12 +1,12 @@
-version: "2.0.0~2"
+version: "3.0.0"
```

Comment:
Are there any breaking changes in this change? If not, we can bump to `2.1.0`

Reply:
The version of IDF has been updated to `>=5.0.1` since `5.0` doesn't support RGB triple-buffer.